_posts/2020-12-02-solidity-underhanded-contest-winners.md

```diff
@@ -0,0 +1,100 @@
+---
+layout: post
+published: true
+title: 'Announcing the Winners of the Solidity Underhanded Contest 👨‍💻🏅'
+date: '2020-12-02'
+author: Franziska Heintel
+category: Announcements
+---
+
+After thorough assessment of all submissions, we are happy to share the winners of this year's [Solidity Underhanded Contest](https://underhanded.soliditylang.org/)!
+
+Before we dive into the winning submissions, we'd like to thank all participants for taking part. In total, we received 16 submissions which you can find in [this repo](https://github.com/ethereum/solidity-underhanded-contest/tree/master/submissions_2020). All 16 submissions are eligible for a "qualified submission" Solidity Underhanded POAP NFT. We will be in touch with participants shortly with details on the claiming process of those.
+
```

Comment:
If we talk about POAPs shoudn't we also talk about the main prizes?

Reply:
Yes fair enough, I'll adjust it.